tests/unit/test_clidriver.py

```diff
@@ -632,13 +632,13 @@ def test_custom_command_schema(self):
         rc = driver.main(
             'ec2 foo --bar Count=4'.split())
 
-        self.assertEqual(rc, 255)
+        self.assertEqual(rc, 252)
 
         # Test extra unknown shorthand item
         rc = driver.main(
```

Comment:
What's your thoughts on return 252 for invalid shorthand syntax as well? I noticed that the PR did not touch the `shorthand.py` module and it has some parsing exceptions: https://github.com/aws/aws-cli/blob/9a897c0c0de4c2c3908edb2d2d4e1efbf1e44d6d/awscli/shorthand.py#L57

Reply:
I'll take a look at the shorthand stuff as well.

Reply:
Looks like our general argprocessing module has a `ParamError` exception (which is used to reraise the shorthand exceptions). I'll add this to the list of param errors we catch.